notifications/listener_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 func TestListener(t *testing.T) {
-	registry := storage.NewRegistryWithDriver(inmemory.New())
+	registry := storage.NewNamespaceWithDriver(inmemory.New())
```

Comment:
I think this is okay to leave as a "registry". A registry effectively serves a namespace, where the namespace is the collection of repositories served by the registry instance (I swear that is not tautological).